pocket_ic/pocket_ic.py

```diff
@@ -62,34 +66,74 @@ def validate(self) -> None:
             or self.sns
             or self.system
             or self.application
+            or self.verified_application
         ):
             raise ValueError("At least one subnet must be configured.")
 
-        if self.application < 0 or self.system < 0:
-            raise ValueError(
-                "The number of application and system subnets must be non-negative."
-            )
```

Comment:
Why is this not needed anymore? What happens if  you multiply a list with a negative number?

Reply:
Then the list is just empty, so the first check suffices

Reply:
Basically validation would fail if there a no special subnets and all subnets where we support multiple of them (verified app, app, system) would be an empty array